src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractLanguageService`2.VsLanguageDebugInfo.cs

```diff
@@ -149,13 +153,17 @@ public int GetProximityExpressions(IVsTextBuffer pBuffer, int iLine, int iCol, i
                             Document document = snapshot.GetOpenDocumentInCurrentContextWithChanges();
                             if (document != null)
                             {
-                                var point = snapshot.GetPoint(iLine, iCol);
-                                var proximityExpressions = _proximityExpressionsService.GetProximityExpressionsAsync(document, point.Position, waitContext.CancellationToken).WaitAndGetResult(waitContext.CancellationToken);
-
-                                if (proximityExpressions != null)
+                                var nullablePoint = snapshot.TryGetPoint(iLine, iCol);
+                                if (nullablePoint.HasValue)
```

Comment:
Should we do this check prior to the snapshot.GetOpenDocumentInCurrentContextWithChanges()? Otherwise we'll be doing a lot of forking for nothing.

Reply:
good idea. thanks! will fix this in the current spot and one below

---
In reply to: [181509476](https://github.com/dotnet/roslyn/pull/26128#discussion_r181509476) [](ancestors = 181509476)